includes/reader-revenue/templates/myaccount-edit-account.php

```diff
@@ -65,7 +67,8 @@ class="woocommerce-Input woocommerce-Input--text input-text"
 	<p class="woocommerce-form-row woocommerce-form-row--wide form-row form-row-wide mt0">
 		<label for="account_email_display"><?php \esc_html_e( 'Email address', 'newspack-plugin' ); ?>
 		<?php if ( $is_email_change_enabled ) : ?>
-		<input type="email" class="woocommerce-Input woocommerce-Input--email input-text" name="account_email" id="account_email" autocomplete="email" value="<?php echo \esc_attr( $user->user_email ); ?>" />
+		<input type="email" class="woocommerce-Input woocommerce-Input--email input-text" name="newspack_account_email" id="newspack_account_email" autocomplete="email" <?php echo \esc_attr( $is_pending_email_change ? 'disabled' : '' ); ?> value="<?php echo \esc_attr( $display_email ); ?>" />
+		<input type="hidden" class="woocommerce-Input woocommerce-Input--email input-text" name="account_email" id="account_email" autocomplete="email" value="<?php echo \esc_attr( $user->user_email ); ?>" />
```

Comment:
This is an edge case for sure, but if you change the value of this hidden input via dev tools, you can change the email and bypass the entire verification flow.

Reply:
Good catch @dkoo! I've addressed this in https://github.com/Automattic/newspack-plugin/pull/3763/commits/f7ea77524123851f655ef0d95b88d4a2b9648cc1.

This commit also handles invalid or existing emails.